tests/test_locality_Voronoi.py

```diff
@@ -5,46 +5,81 @@
 import util
 
 
-@util.skipIfMissing('scipy.spatial')
+def sort_rounded_xyz_array(arr, decimals=4):
```

Comment:
Add a comment on the reason for the sort order.

Reply:
Done.